ci/script.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+source ./tempEnv.sh
+
+set -eEuo pipefail
+
+echo "=== Local variables ==="
+PROD_RELEASE_BRANCH="${PROD_RELEASE_BRANCH:-main}"
+DEV_RELEASE_BRANCH="${DEV_RELEASE_BRANCH:-development}"
+DOCKER_IMAGE_NAME="${DOCKER_IMAGE_NAME:-horizenlabs/blockscout-frontend}"
+
+# absolute path to project from relative location of this script
+workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
+echo "=== Workdir: ${workdir} ==="
+
+command -v docker &> /dev/null && have_docker="true" || have_docker="false"
+echo "=== Have docker: ${have_docker} ==="
+
+# Functions
+function fn_die() {
+  echo -e "$1" >&2
+  exit "${2:-1}"
+}
+
+echo "=== Checking if Docker is installed ==="
+if [ "${have_docker}" = "false" ]; then
+  fn_die "Docker is not installed. Exiting ..."
+fi
+
+echo "=== Checking if GITHUB_TOKEN is set ==="
+if [ -z "${GITHUB_TOKEN:-}" ]; then
+  fn_die "GITHUB_TOKEN variable is not set. Exiting ..."
+fi
+
+echo "=== Checking if DOCKER_WRITER_PASSWORD is set ==="
+if [ -z "${DOCKER_WRITER_PASSWORD:-}" ]; then
+  fn_die "DOCKER_WRITER_PASSWORD variable is not set. Exiting ..."
+fi
+
+echo "=== Checking if DOCKER_WRITER_USERNAME is set ==="
+if [ -z "${DOCKER_WRITER_USERNAME:-}" ]; then
+  fn_die "DOCKER_WRITER_USERNAME variable is not set. Exiting ..."
+fi
+
+docker_tag=""
+if [ "${IS_A_RELEASE}" = "true" ]; then
+  docker_tag="${TRAVIS_TAG}"
```

Comment:
This is redundant

Reply:
I'm sorry @yurikoinaba , but can you elaborate? I'm not seeing why it's redundant, we first assign `""` to `docker_tag`, and then, depending on the branch/release status, we assign it to something else. 
Then later on L60, if `docker_tag` is still `""`, that means is a feature branch and no image is generated, otherwise we continue with the building and publishing process.